packages/swarm-docs/src/pages/index.js

```diff
@@ -4,8 +4,8 @@ import Layout from '../components/layout'
 
 const IndexPage = () => (
   <Layout>
-    <h1>Hi people</h1>
-    <p>Meetup's System in testing for travis</p>
+    <h1 className="text--display">Hello</h1>
+    <p className="text--body">Meetup's System in testing for travis</p>
```

Comment:
if possible, I would recommend renaming this file to `index.jsx` in order for syntax highlighting to work a bit better